android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -144,31 +236,294 @@ private String getCurrentCountry() {
     return current.getCountry();
   }
 
-  private Boolean isEmulator() {
+  private boolean isSupportTelePhony() {
+    PackageManager packageManager = reactContext.getPackageManager();
+    boolean isSupport = packageManager.hasSystemFeature(PackageManager.FEATURE_TELEPHONY);
+    return isSupport;
+  }
+
+  private boolean checkPhoneNumber() {
```

Comment:
I'd like for these names to be more obvious, all these 'check' names are really checking for emu values right? If they were hasEmuPhoneNumber, hasEmuImsi, hasEmuOperatorName, hasEmuDeviceId etc then they wouldn't need javadoc to explain, as it is "checkPhoneNumber" would need a javadoc because you have to explain checking for what type, and under what conditions it returns true or false

Reply:
I renamed as you advice...
https://github.com/react-native-community/react-native-device-info/pull/721/commits/72aa24ded9aa61ae5d380f3138b4928d9bbab7d0